astronomer/airflow/version_check/plugin.py

```diff
@@ -78,6 +88,59 @@ def start_update_thread(cls):
         AstronomerVersionCheck.ensure_singleton()
         CheckThread().start()
 
+    @classmethod
+    def migrate_db_tables(cls):
+        """Apply migrations to the DB tables."""
+        with create_session() as session:
+            cls.create_columns_for_migration(session)
+
+    @classmethod
+    def create_columns_for_migration(cls, session):
+        """Create columns for the migration."""
+        from .models import AstronomerAvailableVersion
+
+        engine = session.get_bind(mapper=None, clause=None)
+        inspector = inspect(engine)
+        if not getattr(inspector, 'has_table', None):
+            inspector = engine
+
+        migrations = {
+            AstronomerAvailableVersion.__tablename__: {
+                'end_of_support': Column('end_of_support', UtcDateTime(timezone=True), nullable=True),
+                'eos_dismissed_until': Column(
+                    'eos_dismissed_until', UtcDateTime(timezone=True), nullable=True
+                ),
+            },
+        }
+        for table_name, columns in migrations.items():
+            existing_columns = [col['name'] for col in inspector.get_columns(table_name)]
+            for column_name, column_obj in columns.items():
+                if column_name not in existing_columns:
+                    cls.add_column(engine, table_name, column_obj)
+
+    @classmethod
+    def add_column(cls, engine, table_name, column_obj) -> None:
+        """Add a column to a table."""
+        try:
+            metadata = MetaData()
+            table = Table(table_name, metadata, autoload_with=engine)
+            if column_obj.name not in [column.name for column in table.columns]:
+                ddl_statement = DDL(
+                    f"ALTER TABLE {table_name} ADD COLUMN {column_obj.name} {column_obj.type.compile(engine.dialect)}"
+                )
+                if not column_obj.nullable:
+                    ddl_statement = DDL(
+                        f"ALTER TABLE {table_name} ADD COLUMN {column_obj.name} {column_obj.type.compile(engine.dialect)} NOT NULL"
+                    )
```

Comment:
Move this to fstring interpolation in the statement above.

Reply:
used alembic now